.eslintrc.json

```diff
@@ -217,6 +217,16 @@
             "skipTemplates": true
           }
         ],
+        "no-restricted-syntax": [
+          "error",
+          {
+            "selector": "CallExpression[arguments.length=1] > MemberExpression.callee > Identifier.property[name='reduce']",
+            "message": "Provide initialValue to .reduce()."
+          },
+          "_FunctionExpression",
+          "_WithStatement",
+          "_BinaryExpression[operator='in']"
```

Comment:
Are these to restrict additional syntax? I don't think we want to ban function expressions.

Reply:
These were additional rules I thought we might want to add in the future. I put them there to easily find them later on

Reply:
I think we should remove these additional rules before merging and rather bring it up on next Dev/Devops meeting if we want to restrict certain syntax in our repo.

Reply:
@AndesKrrrrrrrrrrr do you want to update this PR so we can finish it?

Reply:
added to my devops discipline meeting list

Reply:
Removed, but not discussed.